ui/docs/manual/design/apis/hawk/authorized-scopes.md

```diff
@@ -34,5 +36,5 @@ support for this functionality.
 certificate used temporary credentials. Just include both fields in the JSON
 object before base64 encoding into the `ext` property.
 
-**Note** the way Hawk works, the `ext` property is covered by the HMAC
-signature. So it's not possible to modify this property on-the-fly.
\ No newline at end of file
+<Warning>The way Hawk works, the `ext` property is covered by the HMAC
+signature. So it's not possible to modify this property on-the-fly.  </Warning>
```

Comment:
There still seems to be spacing after the end of the sentence dot.

Reply:
Looks like I'm habitual of adding spacing. I will take care of it.  
Thanks!

Reply:
Hmm, still there. Do you see it [here](https://github.com/taskcluster/taskcluster/pull/1255/files#diff-4736f88ce5aadb2fbbb2526b47ed8623)?